modules/home/assets/js/components/home-screen.js

```diff
@@ -1,17 +1,24 @@
 import { Container, Box } from '@elementor/ui';
-
 import TopSection from './top-section';
+import PromotionBar from './side-bar-promotion';
 
 const HomeScreen = ( props ) => {
 	return (
 		// Box wrapper around the Container is needed to neutralize wp-content area left-padding
 		<Box sx={ { pr: 1 } }>
-			<Container disableGutters={ true } maxWidth="lg" sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, py: { xs: 2, md: 6 } } }>
+			<Container disableGutters={ true } maxWidth="lg" sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, pt: { xs: 2, md: 6 }, pb: 2 } }>
 				<TopSection
 					topData={ props.homeScreenData.data.top }
 					createNewPageUrl={ props.homeScreenData.createNewPageUrl }
 				/>
 			</Container>
+			<Container disableGutters={ true } sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between' } }>
```

Comment:
My idea was to put all containers inside the top container.
Currently I don't think the gap inside the top container serves a purpose.

Reply:
fixed

Reply:
@elchugreeva Can this be inside the top container? That was my original intention. Is there an issue of having the containers inside each other?

Reply:
fixed